### DISCUSSION TITLE
Remove console.log from processHaywardStar

### DISCUSSION THREAD

**Discussion by Crewski** [tagyoureit/nodejs-poolController#522] [Nixie] [answered]
On the next commit, can you please comment out the console.log line from processHaywardStar in PumpStateMessages.ts?  It causes my PM2 logs (and replays) to grow at an astronomical rate because it is logging for every heartbeat for the pump.  It's not tied to the logger so I can't disable it.  I've already changed my local code so no rush, but it would be great to update here whenever you get a chance.  Thanks.

```
    public static processHaywardStar(msg: Inbound) {
        //             src   act   dest             
        //[0x10, 0x02, 0x00, 0x0C, 0x00][0x00, 0x62, 0x17, 0x81][0x01, 0x18, 0x10, 0x03]
        //[0x10, 0x02, 0x00, 0x0C, 0x00][0x00, 0x2D, 0x02, 0x36][0x00, 0x83, 0x10, 0x03] -- Response from pump
        let ptype = sys.board.valueMaps.pumpTypes.transformByName('hwvs');
        let address = msg.source + 96;
        console.log({ src: msg.source, dest: msg.dest, action: msg.action, address: address });  <----  COMMENT OUT
```

**Comment by rstrouse** [comment, marked as answer]
Done
